--- ifmon-2.2.1/Makefile.orig	2009-04-08 09:00:00.000000000 +0200
+++ ifmon-2.2.1/Makefile	2014-02-02 16:56:54.878541883 +0100
@@ -37,7 +37,7 @@
 #
 all config:
 	@for subdir in $(ALL_SRC_DIRS); do			\
-            $(MAKE) -s -C $$subdir -f Makefile $@ || exit 1;	\
+            $(MAKE) -C $$subdir -f Makefile $@ || exit 1;	\
         done
 	@echo "To install, type \"su\" (to gain root privileges)\
 and then type \"make install\"."
@@ -47,7 +47,7 @@
 #
 install:
 	@for subdir in $(ALL_SRC_DIRS); do			\
-            $(MAKE) -s -C $$subdir -f Makefile $@ || exit 1;	\
+            $(MAKE) -C $$subdir -f Makefile $@ || exit 1;	\
         done
 	@echo "Installation done."
 
@@ -56,5 +56,5 @@
 #
 clean:
 	@for subdir in $(ALL_SRC_DIRS); do			\
-            $(MAKE) -s -C $$subdir -f Makefile $@;		\
+            $(MAKE) -C $$subdir -f Makefile $@;		\
         done
--- ifmon-2.2.1/ifmon/Makefile.orig	2009-04-08 09:00:00.000000000 +0200
+++ ifmon-2.2.1/ifmon/Makefile	2014-02-02 16:57:44.751873123 +0100
@@ -51,11 +51,9 @@
 OBJ_C	= $(SRC_C:.c=.o)
 OBJ_CPP	= $(SRC_CPP:.cpp=.o)
 .c.o:
-	@echo "Compiling module $*.o"
-	@+$(CC) -c $*.c $(INC_DIRS) $(CFLAGS)
+	$(CC) -c $*.c $(INC_DIRS) $(CFLAGS)
 .cpp.o:
-	@echo "Compiling module $*.o"
-	@+$(CPP) -c $*.cpp $(INC_DIRS) $(CFLAGS) $(CPPFLAGS)
+	$(CPP) -c $*.cpp $(INC_DIRS) $(CFLAGS) $(CPPFLAGS)
 
 
 $(BIN): prebuild config modules postbuild
